src/managers/KeybindManager.cpp

```diff
@@ -1941,7 +1941,8 @@ void CKeybindManager::pinActive(std::string args) {
 }
 
 void CKeybindManager::mouse(std::string args) {
-    const auto TRUEARG = args.substr(1);
+    const auto TRUEARG = args.substr(1, args.find(' ') - 1);
+    const auto EXTRARG = args.substr(args.find(' ') + 1);
```

Comment:
this might act wonky if we get a `std::string::npos`. Might be worth checking if there exists a space, or even better, using a CVarList